src/bp-messages/classes/class-bp-messages-thread.php

```diff
@@ -1043,6 +1043,9 @@ public static function get_threads_for_user( $args = array() ) {
 		$sql['where']  = "WHERE {$where_sql} {$meta_query_sql['where']}";
 		$sql['misc']   = "GROUP BY m.thread_id {$having_sql} ORDER BY date_sent DESC {$pag_sql}";
 
+		$sql['where'] = apply_filters( 'bp_recipients_recipient_get_where_conditions', $sql['where'], $r );
```

Comment:
@janakkaneriya007 Add comment for a filter like we as added for other

Reply:
@janakkaneriya007 Filter name should be `bp_messages_recipients_get_where_conditions `